ui/packages/app/src/project_setting/form/Labels.js

```diff
@@ -62,57 +82,90 @@ export const Labels = ({ onChange }) => {
     };
   };
 
+  const [labelError, setLabelError] = useState("");
+  const onChange = labels => {
+    const labelsValid =
+      labels.length === 0
+        ? true
+        : labels.reduce((labelsValid, label) => {
+            return labelsValid && label.isKeyValid && label.isValueValid;
+          }, true);
+    setIsValidLabels(labelsValid);
+    if (!labelsValid) {
+      setLabelError(
+        "Invalid labels. Both key and value of a label must contain only lowercase alphanumeric and dash (-) and must start and end with an alphanumeric character"
```

Comment:
Can we add a comma to split this really long sentence? 😅 "Both key and value of a label must contain only lowercase alphanumeric and dash (-)>>>**,** <<< and must start and end with an alphanumeric character"